source/js/slider.js

```diff
@@ -2,43 +2,40 @@ import Splide from '@splidejs/splide';
 import VideoControls from './helpers/video';
 
 const SLIDER_ITEM = 'c-slider__item';
-const ATTR = 'js-slider';
-const BTN = 'js-slider-btn';
-const INDEX = 'js-slider-index';
-const INNER = 'js-slider-inner';
 const AUTOSLIDE = 'js-slider__autoslide';
-const REPEAT = 'js-slider-repeat';
 
 export default class Slider {
     constructor(slider) {
+        const autoPlay = parseInt(slider.getAttribute(AUTOSLIDE));
         this.sliderElement = slider;
         this.splide = new Splide(slider, {
             type: 'loop',
             autoWidth: true,
             focus: 'center',
+            autoplay: !!autoPlay,
+            interval: !!autoPlay ? autoPlay * 1000 : 5000,
```

Comment:
wouldn't it be safer and look more neat if we used the actual Boolean type? 

```suggestion
            autoplay: Boolean(autoPlay),
            interval: Boolean(autoPlay) ? autoPlay * 1000 : 5000,
```